### PR TITLE
Fixed NullPointerException

### PR DESCRIPTION
**Pull request by Nansess**
Caused by: java.lang.NullPointerException: Cannot invoke "java.lang.Boolean.booleanValue()" because the return value of "com.sedmelluq.discord.lavaplayer.tools.JsonBrowser.as(java.lang.Class)" is null

womp womp deezer

### CODE REVIEW

File: main/src/main/java/com/github/topi314/lavasrc/deezer/DeezerAudioSourceManager.java [hunk 1]
```diff
@@ -302,7 +302,8 @@ private List<AudioTrack> parseTracks(JsonBrowser json, boolean preview) {
 	}
 
 	private AudioTrack parseTrack(JsonBrowser json, boolean preview) {
-		if (!json.get("readable").as(Boolean.class)) {
+		Boolean readable = json.get("readable").as(Boolean.class);
+		if (readable == null || !readable) {
```
Comment (topi314) [inline main/src/main/java/com/github/topi314/lavasrc/deezer/DeezerAudioSourceManager.java:306]: should use `JsonBrowser#asBoolean` tbh

**Comment by topi314** [comment]
can you also fix the other occurrence of `json.get("readable").as(Boolean.class)`?

**Comment by topi314** [comment]
maybe this should default to true?

**Comment by Nansess** [comment]
> maybe this should default to true?

honestly yeah

**Comment by topi314** [comment]
both please??

**Review by topi314 — APPROVED** [review]

**Pull request merged**